app/core/config.py

```
@@ -0,0 +1,136 @@
+        if isinstance(v, str):
+            # Parse comma-separated string from environment variable
+            origins = [origin.strip() for origin in v.split(",") if origin.strip()]
+            # Prevent wildcard in non-development environments
+            if "*" in origins:
+                import os
+                env = os.getenv("ENVIRONMENT", "production").lower()
+                if env in ["production", "prod", "staging"]:
+                    raise ValueError(
+                        "CORS wildcard '*' is not allowed in production or staging. "
+                        "Use explicit origins instead."
+                    )
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)